source/docs/yearly-overview/known-issues.rst

```diff
@@ -21,6 +21,20 @@ Invalid build due to missing GradleRIO
 
 Delete your Gradle cache located under ``~$USER_HOME/.gradle``. Windows machines may need to enable the ability to `view hidden files <https://support.microsoft.com/en-us/windows/view-hidden-files-and-folders-in-windows-10-97fbc472-c603-9d90-91d0-1166d1d9f4b5>`__. This issue has only shown up on Windows so far. Please `report <https://github.com/wpilibsuite/frc-docs/issues/new>`__ this issue if you get it on an alternative OS.
 
```

Comment:
I would make the title more generic something like compilation failure on M1 mac or something like that.